src/DocumentationBrowserViewExtension/Docs/Deprecated.html

```diff
@@ -19,7 +19,7 @@
 <style>
 body {
     color: white;
-    font-size: 20px;
+    font-size: 14px;
```

Comment:
Have you changed the default over here?

Reply:
yes, I changed the default font size on all pages.